### DISCUSSION TITLE
Using Figure with Caption

### DISCUSSION THREAD

**Discussion by Nathan269** [awcodes/filament-tiptap-editor#101] [Q&A]
I have noticed that the returned html for adding an image is just a `img` tag with a `title` and `alt` attribute when the file has this data in the Curator library. However, when the caption filed is used in Curator, it is omitted from the HTML generated in TipTap.

Is it possible to use a `figure` with `img` and a `figcaption`  when a caption is present in the Curator media item?

**Comment by awcodes** [comment]
I'll probably add it at some point, but see [https://github.com/awcodes/filament-tiptap-editor/discussions/100](https://github.com/awcodes/filament-tiptap-editor/discussions/100).
